Get-DevConsole/InstallDevConsole.psm1

```diff
@@ -0,0 +1,379 @@
+{
+    DisplayInProgressOperation -text "Searching for previous version"
+    $latestBuild = Get-DevConsoleBuildShortFileName -devConsoleBuildFullFileName $devConsoleBuildFullFileName
+    $currentBuild = Get-IsFirstSetup -currentInstallationDirectory $currentInstallationDirectory
+
+    try
+    {
+        if ($latestBuild -eq $currentBuild.FileVersion)
+        {
+            DisplayCompletedOperation -color Green -text "Update not needed"
+            break
+        }
+        else 
+        {
+            DisplayCompletedOperation -color Green -text "Update needed"
+            $IsFirstSetup = $false
+        }
+    }
+    catch
+    {
+        $IsFirstSetup = $true
+    }   
+    return $IsFirstSetup    
+}
+
+################################################################################
+# Create folder structure to store DevConsole data
+################################################################################
+Function Add-FolderStructure ($directoryName)
+{
+    try 
+    {
+        if (-not (Test-Path -Path $directoryName))
+        {
+            New-Item -ItemType Directory -Path $directoryName > $null
+        }  
+    }
+    catch 
+    {
+        Write-Message -color Red -message "Failed to create DevConsole folder structure"
+        Write-Message -color Red -message "Error: $_"
+    }
+       
+}
+
+################################################################################
+# Terminate DevConsole process
+################################################################################
+Function Stop-DevConsole()
+{
+    try
+    {    
+        Get-process | Where-Object {$_.ProcessName -eq 'DevConsole'} | Stop-Process -Force
+    }
+    catch
+    {
+        Write-Message -color Red -message "Failed to stop active DevConsole session"
+        Write-Message -color Red -message "Error: $_"
+        return $false  
+    }
+}
+
+################################################################################
+# Install latest version of the DevConsole
+################################################################################
+Function Start-InstallDevConsole ($devConsoleBuildFullFileName, $currentInstallationDirectory, $temporaryDownloadDirectory)
+{
+    try
+    {
+        Stop-DevConsole
+        Expand-Archive -Path $("$temporaryDownloadDirectory\$devConsoleBuildFullFileName") -DestinationPath "$currentInstallationDirectory" -Force
+    }
+    catch
+    {
+        Write-Message -color Red -message "Failed to install the DevConsole"
+        Write-Message -color Red -message "Error: $_"
+        retrun $false  
+    }
+
+    return $true
+}
+
+################################################################################
+# Move current installation directory to old cache directory
+################################################################################
+Function Move-CurrentBuildDirectory($isFirstSetup)
+{
+    try
+    {
+        #Move current build to old cache

```

Comment:
Need a space between "#" and the comment.

Reply:
Fixed